src/transaction.rs

```diff
@@ -33,7 +33,7 @@ pub struct TransactionBuilder<T: Transport> {
     /// to locally signing on the node with the default acount.
     pub from: Option<Account>,
     /// The receiver of the transaction.
-    pub to: Address,
+    pub to: Option<Address>,
```

Comment:
Why this change? What happens if to is None? Is that valid?

Reply:
When doing JSON RPC calls to Ganache, sending request `{ to: "0x0..0" }` makes contract creation fail while `{ to: null }` works. :face_with_head_bandage: